plasma_framework/test/helpers/positions.js

```diff
@@ -1,10 +1,19 @@
 const BLOCK_OFFSET = 1000000000;
 const TX_OFFSET = 10000;
+const BLOCK_OFFSET_FOR_TX_POS = 1000000000 / 10000;
```

Comment:
I do not understand this constant and if I did understand I would rather write it as 
`... = BLOCK_OFFSET / TX_OFFSET`

Reply:
aw, you're totally right. This was style from copy pasting contract code as TxPos and UTXOPOS is different lib.